_extensions/seedcase-theme/theme.scss

```diff
@@ -1,12 +1,9 @@
 /*-- scss:defaults --*/
 
-@import url('https://fonts.googleapis.com/css2?family=Noto Sans');
-@import url('https://fonts.googleapis.com/css2?family=Fira+Code');
-@import url('https://fonts.googleapis.com/css2?family=Maven+Pro');
+@import url('https://fonts.bunny.net/css?family=poppins:400roboto-mono:400');
```

Comment:
Ah, my mistake. You're only loading in poppins here, not Roboto-mono. At least that's what the URL sends to.

Reply:
It needs to be two imports, not one.